src/bin/s3Scan.ts

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+import { Command } from 'commander'
+import * as figlet from 'figlet'
+import { LIB_VERSION } from '../version'
+import { S3Scan, S3_SCAN_HTTPS_AGENT } from '../lib'
+
+const program = new Command()
+console.log(figlet.textSync(`S3 Scan ${LIB_VERSION}`))
```

Comment:
As I mentioned before, I think this is unnecessary.

Reply:
yeah forgot to remove it from here